### PR TITLE
Added the possibily to put the UnicornSharedSecret in a file on the

### PR DESCRIPTION
**Pull request by yannisgu**
target system instead of Bob.config

### CODE REVIEW

**Review by retohugi — APPROVED** [review]
Thanks! Lgtm

**Comment by retohugi** [comment]
Btw: still need to look into failing builds. It's the docs again...

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 5.2
+* Add possibilty to store the Unicorn shared secret in a file on the target system, instead of Bob.config
```
Comment (ernstjoss) [inline docs/CHANGELOG.md]: possibility <-> possibilty

File: docs/README.md [hunk 2]
```diff
@@ -45,3 +45,9 @@ where `r` represents the repository name, `g` is a group, `a` is an artifact, `e
 Before using one of the `*-LB*` cmdlet, you need to connect to the load balancer. This can be achieved by running the following command:
 
     Initialize-F5.iControl -HostName $BigIPHostname -Username $Username -Password $Password
+
+## Unicorn Authentication
+So that Lofty can execute an Unicorn synchronisation, it must authenticate it against Unicorn. 
+To do so there is a UnicornSharedSecret. The UnicornSharedSecret can either be configured directly in Bob.config with the `<UnicornSharedSecret>` key
+or alternatively the `UnicornSharedSecretFile` key can be used. This file needs to contain a path to a plain-text fail containing the key.
```
Comment (ernstjoss) [inline docs/README.md]: "This file needs to contain a path to a plain-text fail containing the key" --> "This key needs to contain a path to a plain-text file containing the key"

File: src/lofty/Install-AppItems.ps1 [hunk 3]
```diff
@@ -68,7 +68,15 @@ function Install-AppItems {
...
+        if($config.UnicornSharedSecretFile -and (Test-Path $config.UnicornSharedSecretFile)) {
+            $sharedSecret = Get-Content $config.UnicornSharedSecretFile
+        }
+
+        if(-not $sharedSecret) {
+            $sharedSecret = $config.UnicornSharedSecret
+        }
+
         if(-not $sharedSecret) {
             Write-Error "You must add the UnicornSharedSecret config key to the Bob.config"
```
Comment (ernstjoss) [inline src/lofty/Install-AppItems.ps1]: You must add the UnicornSharedSecretFile or UnicornSharedSecret config key to...

**Review by ernstjoss — APPROVED** [review]
Put in some typo corrections. Over all thx for your effort, I think this is a great enhancement!

**Pull request merged**